src/abstract-register.ts

```diff
@@ -0,0 +1,85 @@
+import { Interfaces, Utils } from "@uns/ark-crypto";
+import { NestedCommandOutput } from "./formater";
+import { CryptoAccountPassphrases } from "./types";
+import { isTokenId } from "./utils";
+import { WriteCommand } from "./writeCommand";
+
+export abstract class AbstractDelegateRegisterCommand extends WriteCommand {
```

Comment:
nit: it's more `AbstractDelegateCommand` because ` DelegateResignCommand extends AbstractDelegateRegisterCommand` is weired